Tabloid/client/src/components/Header.js

```diff
@@ -28,6 +29,9 @@ export default function Header({ isLoggedIn }) {
                 <NavLink tag={RRNavLink} to="/">Home</NavLink>
               </NavItem>
             }
+               <NavItem>
+                <NavLink tag={RRNavLink} to="/Tags" >Tag Managment</NavLink>
```

Comment:
Make sure to keep your casing consistent... you've got pascal casing here, but in your app views module you've got lowercase